operator/helper/duration.go

```diff
@@ -72,6 +72,10 @@ func (d *Duration) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return err
 }
 
+func (d *Duration) UnmarshalText(text []byte) error {
```

Comment:
Similar to comment on `bytesize_test.go`, we should enhance tests to ensure unmarshaling is the same between yaml and mapstructure in `duration_test.go`.